bootinfoscript

```diff
@@ -1681,7 +1681,7 @@ stage2_loc () {
   for HI in ${!HDName[@]}; do
     hdd=${HDName[${HI}]};
 
-    if [ ${offset} -lt  ${HDSize[HI]} ] ; then
+    if [ ${offset} -lt  ${HDSize[$HI]} ] ; then
```

Comment:
What is the actual problem you are trying to solve? ${foo[bar]} and ${foo[$bar]} are equivalent.